### DISCUSSION TITLE
AndroidQQ9.0.90版本侧滑栏精简已失效

### DISCUSSION THREAD

**Discussion by Shiroha1768** [cinit/QAuxiliary#1146] [Q&A] [answered]
![Screenshot_2024-08-23-12-51-49-20_9d26c6446fd7bb8e41d99b6262b17def.jpg](https://github.com/user-attachments/assets/69931761-1776-4bd5-9fb8-3d83090a31d2)

![Screenshot_2024-08-23-12-53-08-88_fc704e6b13c4fb26bf5e411f75da84f2.jpg](https://github.com/user-attachments/assets/0090e771-07a7-4e5e-b76b-585a14e2788a)

![Screenshot_2024-08-23-12-51-59-64_9d26c6446fd7bb8e41d99b6262b17def.jpg](https://github.com/user-attachments/assets/23ce3ea4-7c1c-40ad-a551-ad65e921b2d5)

**Comment by HdShare** [comment, marked as answer]
尝试最新ci-r2405版本，而非正式版
